src/EncompassRest/Contacts/Contact.cs

```diff
@@ -0,0 +1,100 @@
+﻿using EncompassRest.Utilities;
+using Newtonsoft.Json;
+using System;
+
+namespace EncompassRest.Contacts
+{
+    public abstract class Contact : IDirty
+    {
+        internal abstract string ApiPath { get; }
+        [JsonIgnore]
+        public EncompassRestClient Client { get; private set; }
+
+        [JsonIgnore]
+        public ContactNotes Notes { get; private set; }
+
+        public Contact(EncompassRestClient client, string contactId)
+        {
+            Preconditions.NotNull(client, nameof(client));
+            Preconditions.NotNullOrEmpty(contactId, nameof(contactId));
+
+            Id = contactId;
+            Initialize(client);
+        }
+
+        [JsonConstructor]
+        public Contact()
+        {
+        }
+
+        internal void Initialize(EncompassRestClient client)
+        {
+            Client = client;
+            Notes = new ContactNotes(client, ApiPath, Id);
+        }
+        private DirtyValue<string> _firstName;
+        public string FirstName { get { return _firstName; } set { _firstName = value; } }
+        private DirtyValue<string> _lastName;
+        public string LastName { get { return _lastName; } set { _lastName = value; } }
+        private DirtyValue<string> _ownerId;
+        public string OwnerID { get { return _ownerId; } set { _ownerId = value; } }
+        private DirtyValue<ContactAccessLevel?> _accessLevel;
+        [EnumOutput(EnumOutput.Integer)]
+        public ContactAccessLevel? AccessLevel { get { return _accessLevel; } set { _accessLevel = value; } }
+        private ContactAddress _currentMailingAddress;
+        public ContactAddress CurrentMailingAddress { get { return _currentMailingAddress ?? (_currentMailingAddress = new ContactAddress()); } set { _currentMailingAddress = value; } }
```

Comment:
Perhaps add `public string BusinessWebUrl { get; set; }`. It's not in the documentation but it's coming over in the json.